src/views/ConsentContinuationOut.vue

```diff
@@ -385,8 +445,18 @@ export default class ConsentContinuationOut extends Vue {
       }
 
       // load Detail Comment, removing the first line (default comment)
-      const comment: string = filing.consentContinuationOut.comment || ''
+      const comment: string = filing.consentContinuationOut.details || ''
       this.detailComment = comment.split('\n').slice(1).join('\n')
+
+      if (filing.consentContinuationOut.courtOrder) {
+        const courtOrder: CourtOrderIF = filing.consentContinuationOut.courtOrder
```

Comment:
Is `filing.consentContinuationOut.courtOrder` typed? If so then courtOrder doesn't need an explicit type.